github/pulls_reviews.go

```diff
@@ -5,7 +5,10 @@
 
 package github
 
-import "time"
+import (
+	"fmt"
+	"time"
+)
 
 // PullRequestReview represents a review of a pull request.
 type PullRequestReview struct {
```

Comment:
Sorry, I mean, "state" can be "ACCEPTED", "DISMISSED", "CHANGES_REQUESTED" or "COMMENTED" for **Rest API** we are working with now, but "state" also can be "approved", "rejected", or "commented" for pull-request **event API**. https://developer.github.com/v3/activity/events/types/#pullrequestreviewevent

PullRequestReview is used here as a field of event API.
https://github.com/google/go-github/blob/7b47f69ea1f1cb428354f0a6361242e9cb2cf63b/github/event_types.go#L338

So, it's not a good idea to remove existed comment, i guess.
Or, we can remove this comment completely and leave users to check GitHub API document, but i'm not sure.

And, there is a typo. `s/ACCEPTED/APPROVED/`